stable/datadog/README.md

```diff
@@ -12,17 +12,35 @@ Kubernetes 1.4+ or OpenShift 3.4+ (1.3 support is currently partial, full suppor
 
 ## Installing the Chart
 
-To install the chart with the release name `my-release`, retrieve your DataDog API key from your [Agent Installation Instructions](https://app.datadoghq.com/account/settings#agent/kubernetes) and run:
+To install the chart with the release name `my-release`, retrieve your Datadog API key from your [Agent Installation Instructions](https://app.datadoghq.com/account/settings#agent/kubernetes) and run:
 
 ```bash
 $ helm install --name my-release \
     --set datadog.apiKey=YOUR-KEY-HERE stable/datadog
 ```
 
-After a few minutes, you should see hosts and metrics being reported in DataDog.
+After a few minutes, you should see hosts and metrics being reported in Datadog.
 
 **Tip**: List all releases using `helm list`
 
+### Enabling the Datadog Cluster Agent ###
+
+Read about the Datadog Cluster Agent in the [official documentation](https://docs.datadoghq.com/agent/kubernetes/cluster/).
+
+Run the following if you want to deploy the chart with the Datadog Cluster Agent.
+You can also specify `clusterAgent.metricsProvider.enabled=true` if you want to enable the External Metrics Server.
```

Comment:
This is already set in the command below. Either remove this line or remove the option in the bash snippet.

Reply:
I think we should keep both, just to stress that this is the specific option that enables the external metrics provider.

Reply:
Fair enough, then we should change the wording.
Right now it reads as `Run the following [...], you can also specify <option> if you want to enable [...]` but the option is already in the snippet.

What about `Note that specifying <option> will enable [...]` instead?